test/cli/test-files.js

```diff
@@ -89,5 +89,13 @@ describe('files', () => {
         ].join('\n'))
       })
     })
+
+    it('get recursively', () => {
+      return ipfs('files get QmUhUuiTKkkK8J6JZ9zmj8iNHPuNfGYcszgRumzhHBxEEU').then((out) => {
+        const directory = path.join(process.cwd(), 'QmUhUuiTKkkK8J6JZ9zmj8iNHPuNfGYcszgRumzhHBxEEU')
+        expect(fs.readdirSync(directory).length).to.be.eql(8)
+        // TODO add assertion on content of files
```

Comment:
@pgte @diasdavid hm, it's not passing for me, getting:

```
  1) cli --all files with daemon off (requiring js-ipfs core directly) get recursively:

      AssertionError: expected 1 to deeply equal 8
      + expected - actual

      -1
      +8

      at ipfs.then (test/cli/test-files.js:96:56)
      at process._tickDomainCallback (internal/process/next_tick.js:129:7)

  2) cli --all files with daemon on (contacting js-ipfs through http-api) get recursively:

      AssertionError: expected 1 to deeply equal 8
      + expected - actual

      -1
      +8

      at ipfs.then (test/cli/test-files.js:96:56)
      at process._tickDomainCallback (internal/process/next_tick.js:129:7)
```

Seems it's only pulling down one file rather than 8...